src/hooks/core/chat/useFetchChannelHook.ts

```diff
@@ -232,13 +283,18 @@ const useFetchChannelHook = () => {
       });
     });
 
-    queue.addJob({
-      label: 'timestamp update',
-      task: async () => {
-        const timestamp = new Date().toISOString();
-        StorageUtils.channelSignedTimeStamps.set(timestamp);
-      }
-    });
+    if (Platform.OS === 'android') {
+      const timestamp = new Date().toISOString();
+      StorageUtils.channelSignedTimeStamps.set(timestamp);
+    } else {
+      queue.addJob({
+        label: 'timestamp update',
+        task: async () => {
+          const timestamp = new Date().toISOString();
+          StorageUtils.channelSignedTimeStamps.set(timestamp);
+        }
+      });
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This block introduces a platform-specific conditional check for updating the timestamp. On Android, it directly updates the timestamp, while on other platforms, it adds this task to a queue for later processing. This seems like an optimization for Android, but it's important to ensure that this doesn't introduce any race conditions or inconsistencies in the application state.

```diff
+     if (Platform.OS === 'android') {
+       const timestamp = new Date().toISOString();
+       StorageUtils.channelSignedTimeStamps.set(timestamp);
+     } else {
+       queue.addJob({
+         label: 'timestamp update',
+         task: async () => {
+           const timestamp = new Date().toISOString();
+           StorageUtils.channelSignedTimeStamps.set(timestamp);
+         }
+       });
+     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->